src/web_app_skeleton/config/email.cr

```diff
@@ -8,7 +8,7 @@ BaseEmail.configure do |settings|
     send_grid_key = send_grid_key_from_env
     settings.adapter = Carbon::SendGridAdapter.new(api_key: send_grid_key)
   else
-    settings.adapter = Carbon::DevAdapter.new
+    settings.adapter = Carbon::DevAdapter.new(print_emails: true)
```

Comment:
I think it makes sense for this, but maybe turned off in test mode? Also, if we add a quick comment above this that's like `set this to false if logs become too noisy` or something? I'm horrible at wording stuff, but that would help. On a side note, I believe Paul may have mentioned that these will go in to breeze. If that's the case, then this could be a temporary deal anyway. In any case, I'm cool with it 👍

Reply:
Ah, yes I was just adding this to Jumpstart and realized that the test output is **not** pretty, so I added an `elsif` for `development` and left the test environment silent!